joomla/joomla-cms
# Add a filter featured to content articles

### Summary of Changes
At the moment, it is not possible to filter for unfeatured articles in the content.
This PR adds a filter "featured" to the articles view

### Testing Instructions
You should have at least one fieatured and at least one unfeatured article. 
Apply the Patch and the list of filters in the Table Options has a new filter "featured".
Apply the filter to your articles.

### Documentation Changes Required
yes .. new filter
